cartography/intel/gcp/compute.py

```diff
@@ -112,6 +136,62 @@ def transform_gcp_instance(instance, project_id, zone_name):
     return instance
 
 
+def transform_gcp_vpcs(vpc_res):
+    """
+    Transform the VPC response object for Neo4j ingestion
+    :param vpc_res: The return data
+    :return: List of VPCs ready for ingestion to Neo4j
+    """
+    vpc_list = []
+    prefix = vpc_res['id']
+    projectid = prefix.split('/')[1]
```

Comment:
[refactor] is it similar to AWS ARN? if so, we could benefit from common method to parse them and get dictionary back. We redo the same logic in other functions also

Reply:
Yeah, a URI/partial URI in GCP is the same concept of an AWS ARN. I see your point on creating a generic function to handle this - and I'm open to more feedback -  but I would rather wait until there's another place in the code where we need to parse VPC URIs before refactoring.  I will add a comment to make this more readable though.